_pages/en/web-services-access.md

```diff
@@ -21,57 +21,99 @@ This Policy applies only to unclassified assets (data, information, code, ..).
+  Evaluate open source options and co-development with other organizations as well; and
+- Review its technology brick criteria by considering the shared responsibility model of SaaS.
 
 ### Next Steps
 
-Should you approve, the IT Strategy team will work with teams in IITB to:
+The IT Strategy team will work with teams in IITB to:
+
+- Ensure the following SaaS (See [Appendix A](#appendix-a---saas-to-enable)) are available and functional on the network for IITB employees;
+- Ensure IITB employees' understanding of document classification and review access rights to data and information;
+- Support an Options Analysis for tools to manage and monitor use of SaaS on the network; and
+- Review with Technical Architecture the criteria and value add of SaaS in technology bricks.
 
-- Ensure the following SaaS (See [Appendix A](#appendix-a---saas-to-unblock)) are available and functional on the network for IITB employees
-- Increase IITB employees' understanding of document classification and review access rights to data and information
-- Perform an options analysis for tools to manage and monitor use of SaaS on the network
-- Work with Technical Architecture in reviewing the criteria and value add for technology bricks for SaaS
+## Appendix A - SaaS to Enable
 
-## Appendix A - SaaS to unblock
+### Instant Messaging (room-based)
 
-### Collaboration
+#### Benefits
+
+- Don't be restricted by Slack free plan of having access only to the 10 000 most recent messages
+- Ability to encrypt communications (can be a good practice even for unclassified information)
 
 |             | URL           | Notes  |
 | ----------- |:-------------:| ------:|
-| Google Docs (Sheets, Slides, Forms) | https://docs.google.com |  |
-| Google Hangouts | https://hangouts.google.com |  |
 | Riot IM | https://riot.im |  |
 | RocketChat | https://open.rocket.chat |  |
-| Slack | https://slack.com | Websockets blocked |
+| Slack (Websockets) | https://slack.com |  |
+
+### Document Editing
+
+#### Benefits
+
+- Read, co-create, review and share documents
+
+|             | URL           | Notes  |
+| ----------- |:-------------:| ------:|
+| Google Docs (Sheets, Slides, Forms) | https://docs.google.com |  |
+
+### Web Conferencing
+
+#### Benefits
+
+- Meet with external stakeholders without the need to install or run a WebEx plugin (with sometimes limited success)
+
+|             | URL           | Notes  |
+| ----------- |:-------------:| ------:|
+| Google Hangouts | https://hangouts.google.com |  |
 | Jitsi Meet | https://meet.jit.si |  |
 
 ### Version Control System (VCS) and Project Management
 
+#### Benefits
+
+- Work on GitHub with SSH keys for better security
+
 |             | URL           | Notes  |
 | ----------- |:-------------:| ------:|
 | Github | https://github.com | Websockets and SSH blocked |
 
 ### Email
```

Comment:
I'm not sure what is implied as an advantage here.

Reply:
SSH keys are more secure than username/password, but I agree this is kinda a moot point if we enable 2FA. How are Websockets used in GitHub (to find benefits in enabling them)?

Reply:
Argh, my bad, I thought I was tagging the email SaaS service with "unique email" for external IT professionals line :man_facepalming: